hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/router/RouterClientProtocol.java

```diff
@@ -819,20 +821,34 @@ public void renewLease(String clientName, List<String> namespaces)
     }
   }
 
+  /**
+   * For {@link this#getListing(String,byte[],boolean)} to sort results.
```

Comment:
This java doc are not compliant with rules. try to as the following style.
`For {@link #getListing(String,byte[],boolean) GetLisiting} to sort results.`